### PR TITLE
Enhance LocalAccountDTO with deletion Info

### PR DESCRIPTION
**Pull request by Siolto**
Still WIP. Please do not review!

FYI: @Milena-Czierlinski

### CODE REVIEW

File: packages/runtime/test/transport/relationships.test.ts
Comment (Milena-Czierlinski) [inline packages/runtime/test/transport/relationships.test.ts:1]: @Siolto This doesn't belong to this PR, does it?

Comment (jkoenig134) [reply packages/runtime/test/transport/relationships.test.ts:1]: I think @Siolto used this to test out whats going on during identity deletion

File: packages/app-runtime/test/modules/DatawalletSynchronizedModule.test.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        await sessionDevice1.transportServices.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        expect(sessionDevice2.account.deletionDate).toBeUndefined();
+
+        await runtimeDevice2.init(); // TODO: do the modules need to be turned on?
+        await runtimeDevice2.start();
+
+        await sessionDevice2.transportServices.account.syncDatawallet();
+        // await TestUtil.awaitEvent(runtimeDevice2, LocalAccountDeletionDateChangedEvent);
+
+        expect(sessionDevice2.account.deletionDate).toBeDefined();
```
Comment (Milena-Czierlinski) [inline packages/app-runtime/test/modules/DatawalletSynchronizedModule.test.ts:88]: @jkoenig134 Here we don't seem to go into the DatawalletSynchronized module. Do you have any idea why?

File: packages/runtime/test/transport/relationships.test.ts
Comment (Siolto) [reply packages/runtime/test/transport/relationships.test.ts:1]: yep I think this can be removed. It was only to figure things out => Delete
